### DISCUSSION TITLE
Power plans & G-Helper

### DISCUSSION THREAD

**Discussion by Konnos998** [seerge/g-helper#1748] [General]
My laptop's battery drains very quickly after some steps and I think it's the Windows Balanced power plan. Is it ok to set the power plan to Asus (recommended) again and see how it goes?

**Comment by seerge** [comment]
@Konnos998 hello, I don't think that windows power plan is related (as far as you use Battery efficient **Power Mode**), but you can enforce any of the power plan you want in app's config

https://github.com/seerge/g-helper?tab=readme-ov-file#custom-windows-power-plans-with-each-mode

**Reply by Konnos998** [reply]
Thank you very much!

**Comment by Konnos998** [comment]
That's an example for the default Windows power plan:
"scheme_0": "381b4222-f694-41f0-9685-ff5bb260df2e",
"scheme_1": "381b4222-f694-41f0-9685-ff5bb260df2e",
"scheme_2": "381b4222-f694-41f0-9685-ff5bb260df2e",

How can I do the same to force the Asus (recommended) power plan? Because if I do it from the power plan settings after some time it changes automatically to Windows again (and I assume G-Helper does it).
![Στιγμιότυπο οθόνης (1200)](https://github.com/seerge/g-helper/assets/128422948/e97a351f-c4e8-47af-83cf-ade75b906428)

Also that's a screenshot of my json file (and even tho I put scheme 0, after I restarted my laptop it's gone automatically):

![Στιγμιότυπο οθόνης (1201)](https://github.com/seerge/g-helper/assets/128422948/27584fc7-cc4b-44de-a3a5-c657b42c5aaa)

**Reply by Konnos998** [reply]
My power plan's code (for Asus, which I found via cmd) is 3355e07a-8410-493b-a9f3-ba15fb7627b3.
But do I put scheme_0, scheme_1 or scheme_2?

**Reply by seerge** [reply]
@Konnos998 it's a setting for silent / balanced / turbo. If you want to use same plan everywhere - put it everywhere

**Reply by Konnos998** [reply]
So I have to put it everywhere on all schemes (0,1 and 2). Do I have to quit ghelper in order for the changes to take effect? Also, I've seen that in my json configuration after : there are no double quotes (" "), so I might have to put the setting without double quotes (like
"scheme_0" : 3355e07a-8410-493b-a9f3-ba15fb7627b3).

**Reply by seerge** [reply]
Yes, you put it everywhere. 

You need to quit app, make changes and start it over.

Both - param name, and value (plan id) should be in quotes

**Reply by Konnos998** [reply]
Got it! I exited the app, I made it to always run it as admin, make it an exclusion on windows defender (just in case), made the changes with quotes and then started it over manually! I think it's ok now because it didn't override any of the changes after I opened it (something that did before, and maybe was due to admin privilages)...If I have any more questions I'll send here, thanks in advance!
